root/etc/e-smith/db/configuration/migrate/Remove_logs_delegation

```diff
@@ -0,0 +1,18 @@
+{
+    #
+    # Remove the `logs` delegation to users  
+    #
+    use esmith::ConfigDB;
+    
+    my $db = esmith::ConfigDB->open();
+    my $newProp = '';
+    my @Keys = split(',', $db->get_prop('cockpit.socket','delegation')|| '');
+
+    foreach (@Keys) {
+        my @props = split(':',$_);
+        my @props = grep { !/logs/ } @props;
+        $newProp .= join(':',@props).',';
```

Comment:
Can we change the current loop item? (see the `join()` below)
```suggestion
        $_ = join(':', $group_name, @props);
```

Reply:
done